LWTrie/LWTrie.m

```diff
@@ -308,6 +308,32 @@ - (void)statsInternal:(NSMutableDictionary *)stats forNode:(LWTrieNode *)node {
     }
 }
 
+- (NSArray *)valuesWithPrefix:(NSString *)prefix{
+
+    NSMutableArray* valueList = [[NSMutableArray alloc]init];
```

Comment:
For consistency with existing code style, please associate the * with the var name, and add a space before init.